src/core/buildDesignTokensJson.ts

```diff
@@ -62,6 +62,16 @@ export const buildRestyleThemeObject = (designTokensJson: any) => {
           themeObject['breakpoints'][key] = `tokens.${groupKey}.${key}`
         })
       }
+    } else if (groupKey.toLocaleLowerCase() === 'figmaeffects') {
+      const effectValues = designTokensJson && designTokensJson[groupKey as keyof unknown]
+      if (!_.isEmpty(effectValues)) {
+        if (!themeObject['shadows']) {
+          themeObject['shadows'] = {}
+        }
+        Object.keys(effectValues).map((key) => {
+          themeObject['shadows'][key] = `tokens.${groupKey}.${key}.offset.x + "px " + ${groupKey}.${key}.offset.y + "px " + tokens.${groupKey}.${key}.radius + "px " + tokens.${groupKey}.${key}.color`
```

Comment:
Thank you for that!

I tested it and it was not generating a unique shadow string properly on the generated theme typescript file, so I used your code and only had to change this part to make it like this:

        Object.keys(effectValues).map((key) => {
          themeObject['shadows'][key] = { 
            color: `tokens.${groupKey}.${key}.color`,
            offsetX: `tokens.${groupKey}.${key}.offset.x`,
            offsetY: `tokens.${groupKey}.${key}.offset.y`,
            radius: `tokens.${groupKey}.${key}.radius`
          }
        })